docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -52,7 +50,10 @@ Follow these steps for a new or existing project to configure it to use a shared
     <DefaultSettings>
     ```
 
-1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
+> [!NOTE]
+> Don't add the shared runtime requirement for a Word add-in. It will cause an error when loading the add-in
```

Comment:
Include the space so the numbering will continue
```suggestion
    > [!NOTE]
    > Don't add the shared runtime requirement for a Word add-in. It will cause an error when loading the add-in
```